plenoptic/simulate/models/naive.py

```diff
@@ -136,7 +136,9 @@ def filt(self):
         if self._filt is not None:  # use old filter
             return self._filt
         else:  # create new filter, optionally cache it
-            filt = circular_gaussian2d(self.kernel_size, self.std, self.out_channels)
+            device = self.std.device
+            filt = circular_gaussian2d(self.kernel_size, self.std, self.out_channels, device)
+            filt = filt.to(device)
```

Comment:
is the `filt.to(device)` necessary with the device arg passed to `circular_gaussian2d`?

Reply:
Good catch. No it is not. Removed.